docs/setup/install/deb.asciidoc

```diff
@@ -115,6 +121,8 @@ sudo dpkg -i kibana-{version}-amd64.deb
 <1> Compare the SHA produced by `shasum` with the
     https://artifacts.elastic.co/downloads/kibana/kibana-{version}-amd64.deb.sha512[published SHA].
 
+You can alternatively download the open source distribution of Kibana from: https://artifacts.elastic.co/downloads/kibana/kibana-oss-{version}-amd64.deb    
```

Comment:
Can we phrase this more like "open source only distribution" rather than simply "open source distribution"?